src/python/pants/base/pants_env_vars.py

```diff
@@ -0,0 +1,8 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+
+IGNORE_UNRECOGNIZED_ENCODING = "PANTS_IGNORE_UNRECOGNIZED_ENCODING"
+RECURSION_LIMIT = "PANTS_RECURSION_LIMIT"
+DAEMON_ENTRYPOINT = "PANTS_DAEMON_ENTRYPOINT"
+PANTSC_PROFILE = "PANTSC_PROFILE"
```

Comment:
Huh, I wonder why it's pantsc and not pantsd.

Reply:
> Huh, I wonder why it's pantsc and not pantsd.

Is it supposed to represent "pants c_profile" perhaps?

Reply:
It's for the client: see https://v1.pantsbuild.org/invoking.html#profiling-pants